<launch>

    <include file="$(find seabot)/launch/seabot_base.launch"/>

    <node name="sliding_node" pkg="seabot_depth_regulation" type="sliding_node" output="screen" ns="regulation" respawn="true" respawn_delay="5">
        <param name="g" value="9.81" />
        <param name="rho_eau" value="1000.0" />
        <param name="m" value="6.855" />
        <param name="C_f" value="0.08" />
        <param name="tick_to_volume" value="1.431715402026599e-07" />

        <param name="frequency" value="1.0" />
        <param name="offset" value="700" />
        <param name="max_delta_tick" value="400" />
        <param name="K_factor" value="10.0" />
        <param name="K_velocity" value="50" />
    </node>

    <node name="regulation_talker" pkg="seabot_depth_regulation" type="regulation_talker.py" output="screen" respawn="true">
    </node>

</launch>

